src/gql/queries/dataset.ts

```diff
@@ -84,6 +90,33 @@ export const GET_LINKABLE_DATASETS = gql`
   ${layerFragment}
 `;
 
+export const IMPORT_DATASET = gql`
+  mutation ImportDataset($file: Upload!, $sceneId: ID!, $datasetSchemaId: ID) {
+    importDataset(input: { file: $file, sceneId: $sceneId, datasetSchemaId: $datasetSchemaId }) {
+      datasetSchema {
+        id
+        name
+      }
+    }
+  }
+`;
+
+export const REMOVE_DATASET = gql`
+  mutation RemoveDataset($schemaId: ID!, $force: Boolean) {
+    removeDatasetSchema(input: { schemaId: $schemaId, force: $force }) {
+      schemaId
+    }
+  }
+`;
+
+export const REMOVE_DATASET_SCHEMA = gql`
+  mutation RemoveDatasetSchema($schemaId: ID!) {
+    removeDatasetSchema(input: { schemaId: $schemaId }) {
+      schemaId
+    }
+  }
+`;
```

Comment:
If you could, can you investigate these a little and see if you can remove one? If the backend for `removeDataseetSchema` has the `force` argument as optional, you could update here and use the one mutation for both (with and without force)!

Reply:
removed RemoveDatasetSchema as it seems no longer in use